src/commands/config/set.js

```diff
@@ -135,21 +128,50 @@ const handlers = {
 	pretty: setBoolean,
 };
 
-export const actionCreator = () => async ({ variable, values }) => {
-	if (!CONFIG_VARIABLES.includes(variable)) {
-		throw new ValidationError('Unsupported variable name.');
+export default class SetCommand extends BaseCommand {
+	async run() {
+		const { args: { variable, values } } = this.parse(SetCommand);
+		const safeValues = values || [];
+		const safeValue = safeValues[0] || '';
+		const result = handlers[variable](
+			this.userConfig,
+			variable,
+			safeValue,
+			safeValues,
+		);
+		this.print(result);
```

Comment:
This still uses the original print options, so changes to print-related config variables won't be immediately reflected.